docker/development/debian/Dockerfile

```diff
@@ -63,11 +63,24 @@ RUN mkdir /tmp/eigen \
 ARG IAU_SOFA_RELEASE="2018_0130_C"
 ARG IAU_SOFA_VERSION="20180130"
 
+#RUN mkdir -p /tmp/sofa \
+# && cd /tmp/sofa \
+# && wget --quiet http://www.iausofa.org/${IAU_SOFA_RELEASE}/sofa_c-${IAU_SOFA_VERSION}.tar.gz \
+# && tar -zxf sofa_c-${IAU_SOFA_VERSION}.tar.gz \
+# && cd sofa/${IAU_SOFA_VERSION}/c/src/ \
+# && pattern="CFLAGF = -c -pedantic -Wall -W -O" \
+# && target="CFLAGF = -c -pedantic -Wall -W -O -fpic" \
+# && sed -i -e 's,'"$pattern"','"$target"',g' makefile \
+# && make -j $(nproc) \
+# && mkdir /usr/local/include/sofa \
+# && cp -r *.h /usr/local/include/sofa \
+# && cp -r *.a /usr/local/lib \
+# && rm -rf /tmp/sofa
+
 RUN mkdir -p /tmp/sofa \
  && cd /tmp/sofa \
- && wget --quiet http://www.iausofa.org/${IAU_SOFA_RELEASE}/sofa_c-${IAU_SOFA_VERSION}.tar.gz \
- && tar -zxf sofa_c-${IAU_SOFA_VERSION}.tar.gz \
- && cd sofa/${IAU_SOFA_VERSION}/c/src/ \
+ && git clone "https://github.com/Starlink/sofa.git" \
```

Comment:
What’s the rationale for this move? (I think it’s better to install from the official source no?)

Reply:
Hey! Yeah this is from testing. Removing this now

The official source for this was actually down the other day, so this was a workaround